noir-projects/noir-contracts/contracts/claim_contract/src/main.nr

```diff
@@ -28,18 +29,24 @@ pub contract Claim {
     }
 
     #[private]
-    fn claim(proof_note: UintNote, note_storage_slot: Field, recipient: AztecAddress) {
+    fn claim(proof_retrieved_note: RetrievedNote<UintNote>, recipient: AztecAddress) {
         // 1) Check that the note corresponds to the target contract and belongs to the sender
         let target_address = storage.target_contract.read();
         assert(
-            target_address == proof_note.header.contract_address,
+            target_address == proof_retrieved_note.contract_address,
             "Note does not correspond to the target contract",
         );
-        assert_eq(proof_note.owner, context.msg_sender(), "Note does not belong to the sender");
+        assert_eq(
+            proof_retrieved_note.note.owner,
+            context.msg_sender(),
+            "Note does not belong to the sender",
+        );
 
         // 2) Prove that the note hash exists in the note hash tree
+        // Note: The note has been inserted into the donation_receipts set in the Crowdfunding contract.
+        let note_storage_slot = Crowdfunding::storage_layout().donation_receipts.slot;
```

Comment:
You can just do `storage.donation_receipts.slot`, no need to access the layout directly.

Reply:
This is a storage layout of a different contract (here we claim a reward in a Claim contract based on a note in the Crowdfunding contract)

Reply:
Ah you're right, silly me.